lte/gateway/python/magma/mobilityd/ip_allocator_dhcp.py

```diff
@@ -290,13 +290,13 @@ def get_dhcp_desc_from_store(
         LOG.debug("lookup error for %s", str(key))
         return None
 
-    def alloc_ip_address(self, sid: str, vlan: int) -> IPDesc:
+    def alloc_ip_address(self, sid: str, vlan_id: int) -> IPDesc:
```

Comment:
implements `IPAllocator.alloc_ip_address(self, sid: str, vlan_id: int)` -> `vlan_id` should not be renamed (or renamed everywhere)